src/app/patient-dashboard/common/procedure-orders/procedure-orders.component.spec.ts

```diff
@@ -0,0 +1,229 @@
+// import { async, ComponentFixture, TestBed, inject } from '@angular/core/testing';
```

Comment:
Do we need to keep this files which are all commented? Probably we should remove them from the source.

Reply:
am working on the unit tests